.github/actions/install_docgen/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Install Docgen
+description: |
+  Install Docgen
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout talos
+      uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # tag=v3.1.0
+      with:
+        fetch-depth: 0
+        repository: "siderolabs/talos"
+        ref: "v1.2.6"
+        path: talos
+
+    - name: Remove go.work
```

Comment:
Maybe a good place to leave a comment that this is required for building docgen.